include/cell/compute/broadcast.hpp

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "cuda_utils.hpp"
+#include "types/layout.hpp"
+#include "types/tile_shape.hpp"
+
+namespace tiledcuda::cell::compute {
+namespace tl = tile_layout;
+
+template <typename SrcTile, typename DstTile, const tl::Layout kLayout>
+struct Broadcast {
+    using DType = typename DstTile::DType::DType;
+
+    static constexpr int kRows = DstTile::kRows;
+    static constexpr int kCols = DstTile::kCols;
+
+    DEVICE void operator()(const SrcTile& src, DstTile& dst) {}
+};
+
+template <typename SrcTile, typename DstTile>
+struct Broadcast<SrcTile, DstTile, tl::Layout::kRowMajor> {
```

Comment:
Does this implementation perform a broadcast assignment? Are broadcast operations like add, subtract, etc., necessary, and do you plan to support them?

Reply:
Incorporating operations like Add and Sub into Broadcast also seems feasible. My original idea was to place them in Elementwise and Binary, and then execute the operations after the broadcasting?

Reply:
It seems we could add a `Functor` template to do this. Taking an extra step doesn't seem to excessively introduce additional code complexity.

Reply:
Oh. I see.